tiles/src/main/java/com/protomaps/basemap/Basemap.java

```diff
@@ -173,9 +194,29 @@ static void run(Arguments args) throws IOException {
           Paths.get(clipArg));
     }
 
+    List<String> renderOnlyOptions = List.of(
+      "Boundaries",
```

Comment:
should these be lower case to match the vector tile names? maybe we could also retrieve them from the layer classes instead of hardcoding?

Reply:
I pull them from the classes now.